generators/app/index.js

```diff
@@ -198,7 +221,7 @@ module.exports = generators.Base.extend({
 
     this.composeWith('node:git', {
       options: {
-        repositoryPath: this.props.repository
+        repositoryPath: 'git@github.com:' + this.props.repository + '.git'
```

Comment:
npm default to github. No need for the full path.

Also, what happens here when the repository field is empty?